PyInstaller/utils/hooks/conda.py

```diff
@@ -121,8 +121,7 @@ def walk_dependency_tree(initial, excludes=None):
             logger.warning(
                 "Conda distribution '%s', dependency of '%s', was not found. "
                 "If you installed this distribution with pip then you may "
-                "ignore this warning." % (name, initial)
-            )
+                "ignore this warning." % (name, initial))
```

Comment:
Please abstain from non-functional changes like this one. See development guideline for rational.